README.md

```diff
@@ -138,7 +138,7 @@ This part shows how you might interact with the middleware during runtime:
 
 In order to develop a server-side rendering application, we need access to the [`stats`](https://github.com/webpack/docs/wiki/node.js-api#stats), which is generated with the latest build.
 
-In the server-side rendering mode, __webpack-dev-middleware__ would sets the `stat` to `res.locals.webpackStats` before invoking the next middleware, where we can render pages and response to clients.
+In the server-side rendering mode, __webpack-dev-middleware__ sets the `stat` to `res.locals.webpackStats` before invoking the next middleware, where we can render pages and responses to clients.
```

Comment:
maybe "the `stat` property"

"where we can render pages and responses to clients." should probably be rewritten. perhaps something like:

"allowing a developer to render the page body and manage the response to clients."